### ISSUE TITLE
Globally does not search for text inside a pdf

### ISSUE THREAD

**Issue by patsevanton** [tadashi-aikawa/obsidian-another-quick-switcher#238]
Hello! Thanks for another-quick-switcher! 
I am use Ubuntu
```
whereis rg
rg: /usr/bin/rg /usr/share/man/man1/rg.1.gz
```
I can search inside PDF
![image](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/assets/10828883/105c899d-834e-4f4f-8a2f-3250b3d5bf66)

Extention - md,pdf
![image](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/assets/10828883/fbbde9f4-c8a5-4451-92b6-d2520ade8f36)

But global not searching
![image](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/assets/10828883/93817b88-9a12-48b3-bb6f-a1831b985eec)

**Comment by tadashi-aikawa** [comment]
Hi, @patsevanton :)

Since ripgrep does not support searching PDFs, AQS also cannot search PDFs.

```
wget https://pdfobject.com/pdf/sample.pdf
rg dolor sample.pdf
# No results
```

**Comment by patsevanton** [comment]
https://github.com/phiresky/ripgrep-all search in PDF

rga condimentum
sample.pdf
Page 1: pharetra commodo, eros mi condimentum quam, sed commodo justo quam ut velit.
Page 1: Phasellus ullamcorper consectetuer ante. Duis tincidunt, urna id condimentum luctus, nibh
Page 1: molestie cursus. Duis ut magna at justo dignissim condimentum. Cum sociis natoque

**Comment by tadashi-aikawa** [comment]
Yes, it is true that ripgrep-all can search PDFs. However, AQS supports only ripgrep, not ripgrep-all.

It might work if you specify the path to the rga executable, but this has not been confirmed and is not guaranteed.
